### ISSUE TITLE
params error

### ISSUE THREAD

**Issue by zhangdan0602** [jisu1013/DGCF_torch#2]
HI, jisu
I find that there are some errors in your code. For example:

the params of nn.Embedding are int, int...But, they are emb,list in the line 228. You can use torch.index_select(tensor, dim, ...) instead of tf.nn.embedding_lookup.
the second param of torch.split() should be the dim that tensor is splited, rather than the num of split.

**Comment by cczq-bjut** [comment]
Hello，zhang
Can you run this code smoothly now？

**Comment by cczq-bjut** [comment]
@zhangdan0602
